docs/interpreter/jdbc.md

```diff
@@ -167,6 +167,10 @@ There are more JDBC interpreter properties you can specify like below.
     <td>default.jceks.credentialKey</td>
     <td>jceks credential key</td>
   </tr>
+  <tr>
+    <td>zeppelin.interpreter.precode</td>
```

Comment:
I agree @felixcheung's opinion. If you want to extend this feature for the future, you can modify it when you do it.s